### ISSUE TITLE
AttributeError: module 'algos' has no attribute 'floyd_warshall'

### ISSUE THREAD

**Issue by 13631818862** [RinneSz/GMAE#4]
Hello, I would like to learn the application of graph on Transformer through your code, but I have encountered this error.
"AttributeError: module 'algos' has no attribute 'floyd_warshall'"
Unfortunately, I cannot find a relevant solution online. I suspect that the import of algos.pyx may not have been successful. Could you tell me how to solve this problem?
Thank you.

**Comment by RinneSz** [comment]
You need to install cython. You can look for the tutorial on how to install cython online.

**Comment by 13631818862** [comment]
Thank you. I will give it a try.

**Comment by 13631818862** [comment]
After reading the installation and use tutorial of cpython, I successfully imported algos.pyx. 

But when I compile algos.pyx using the self-built setup.py file on windows, the following error message will appear:
"ValueError: Buffer dtype mismatch, expected 'long' but got 'long long'"
The displayed error is caused by this code: "cdef numpy.ndarray[long, ndim=2, mode='c'] path = numpy.zeros([n, n], dtype=numpy.int64)"

Therefore, I have made the following changes(long -> int64_t).
from libc.stdint cimport int32_t, int64_t
cdef numpy.ndarray[int64_t, ndim=2, mode='c'] path = numpy.zeros([n, n], dtype=numpy.int64)

In short, I would like to ask whether this change will affect the results of the model. Thank you again.

**Comment by RinneSz** [comment]
I do not think this will change the result, but I would suggest you to run the code on linux if possible. I did not test it on windows, so there might be some unexpected errors.

**Comment by 13631818862** [comment]
OK. Thank you for your suggestion.
